docs/formats.md

```diff
@@ -163,8 +163,8 @@ Name the map by adding a 'mapName' attribute on the file object in your config.
 **Example**  
 ```scss
 $tokens: (
-  $color-background-base: #f0f0f0;
-  $color-background-alt: #eeeeee;
+  'color-background-base': #f0f0f0;
```

Comment:
Sorry this was unclear, this markdown file is being generated by a build process. To make this change, update the JSDoc comment in the `lib/common/formats.js` file and run `npm run generate-docs`. Thanks!

Reply:
No problem Danny :wink: